Products/Zuul/catalog/model_catalog.py

```diff
@@ -149,7 +149,7 @@ def _get_forbidden_classes(self):
         return ()
 
     def get_indexes(self):
-        return self._data_manager.get_indexes()
+        return self.model_index.get_indexes()
```

Comment:
I would leave the call to _data_manager in case data manager adds any additional behavior to `get_indexes ` in the future. We want to isolate calls to modelindex as much as possible, as it is a external package.